tests/test_search/test_elastic.py

```diff
@@ -843,6 +846,24 @@ def test_multiple_migrations_with_delete(self):
             assert_equal(var[settings.ELASTIC_INDEX + '_v{}'.format(n + 1)]['aliases'].keys()[0], settings.ELASTIC_INDEX)
             assert not var.get(settings.ELASTIC_INDEX + '_v{}'.format(n))
 
+    def test_migration_institutions(self):
+        migrate(delete=True, index=settings.ELASTIC_INDEX, app=self.app.app)
+        count_query = {}
+        count_query['aggregations'] = {
+            'counts': {
+                'terms': {
+                    'field': '_type',
+                }
+            }
+        }
+        res = self.es.search(index=settings.ELASTIC_INDEX, doc_type=None, search_type='count', body=count_query)
+        for bucket in res['aggregations']['counts']['buckets']:
+            if bucket['key'] == u'institution':
+                assert True
```

Comment:
Better, but we lean towards using nose asserts rather than bare asserts. If you set `institution_bucket_found = False` above this loop, you can `assert_true(institution_bucket_found)` after. This will give a better error if tests go wrong.